src/client/datascience/utils.ts

```diff
@@ -18,7 +19,7 @@ export async function calculateWorkingDirectory(
     let workingDir: string | undefined;
     // For a local launch calculate the working directory that we should switch into
     const settings = configService.getSettings(resource);
-    const fileRoot = settings.notebookFileRoot;
+    const fileRoot = untildify(settings.notebookFileRoot);
```

Comment:
Not ultra important, but a quick fix for an issue that I ran into myself. Without this ~/Documents registers (correctly) as non-absolute and it tries to append it onto the current working path as opposed to using it by itself.